manifests/rhoai/shared/apps/pachyderm/pachyderm-app.yaml

```diff
@@ -47,7 +47,7 @@ spec:
 
     Before you start the operator installation process, you will need to:
 
-    - Familiarize yourself with Pachyderm's **[main concepts](https://docs.pachyderm.com/products/mldm/latest/learn/basic-concepts/)**, and Pachyderm's command-line tool `pachctl`. You will interact with Pachyderm using `pachctl` from the cells of your notebooks. Read through our written demo [Open CV](https://docs.pachyderm.com/products/mldm/latest/get-started/beginner-tutorial/) to get used to the main commands.
+    - Familiarize yourself with Pachyderm's **[main concepts](https://github.com/pachyderm/pachyderm?tab=readme-ov-file#getting-started)**, and Pachyderm's command-line tool `pachctl`. You will interact with Pachyderm using `pachctl` from the cells of your notebooks. Read through our written demo [Open CV](https://github.com/pachyderm/pachyderm?tab=readme-ov-file#getting-started) to get used to the main commands.
```

Comment:
This is the link for basic concepts https://github.com/pachyderm/docs-content/blob/main/latest/learn/basic-concepts.md. Looks like they are templates for the website because there are some placeholder strings. However, the website is down. 😔